doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,95 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron holography data:
+ 
+* :py:class:`~._signals.hologram_image.HologramImage`
+* :py:class:`~._signals.wave_image.ComplexSignal2D`
+
+Both inherit directly from the :py:class:`~._signals.signal2d.Signal2D` class and thus can use all of
+its functionality. The usage of both classes is explained in the following sections.
```

Comment:
This is also not true anymore, `ComplexSignal2D` and `Signal2D` are now parallel in the inheritence tree (see the diagram I added).